launchable/test_runners/bazel.py

```diff
@@ -28,8 +29,9 @@ def subset(client):
 
 
 @click.argument('workspace', required=True)
+@click.option('--build-event-json', 'build_event_json', help="set file path generated by --build_event_json_file", type=str, required=False)
```

Comment:
`type=click.Path(exists=True)` would ensure this is a file that exists.

Reply:
TIL, cool